### ISSUE TITLE
Flatpak/flathub support?

### ISSUE THREAD

**Issue by pollux78** [WootingKb/wootility-issues#283]
**Is your feature request related to a problem? Please describe.**

I see that you use appimage and thats great but i would definitely check out flatpak, as flatpak supports every single distro and is used by just about everyone in the linux community + it's preinstalled on lots of Distros.

**Describe the solution you'd like**

https://flathub.org/

Flatpak is containerized meaning it uses its own dependencies so that it can work across every distro and so the app doesn't break, similar to how appimage works but the cool thing about flatpak is that 

1. They have a website store called flathub where users can easily get new applications 
2. The repo they use is supported on every major desktop environment gui store like kde plasma, gnome, cosmic, cinnamon

# Documentation 

https://docs.flatpak.org/en/latest/
publishing.html

# Flatpak building docs

https://docs.flatpak.org/en/latest/first-build.html

## Searching for the razer genie through the flathub repo

![Screenshot_20240516_230734](https://github.com/WootingKb/wootility-issues/assets/81181814/708be58d-a603-407e-b6a3-b1f1b392bc41)

## Screenshot of what it looks like to install a Flatpak on a gui store

![Screenshot_20240516_230738](https://github.com/WootingKb/wootility-issues/assets/81181814/2dda7925-cc9a-4e14-9d73-ba1c2c730add)

## Flathub website

![Screenshot_20240516_230850](https://github.com/WootingKb/wootility-issues/assets/81181814/f3b6e7ba-986d-4d3f-8ff9-cfe607870c35)

## They also show recently updated apps

![Screenshot_20240516_230945](https://github.com/WootingKb/wootility-issues/assets/81181814/61c94c20-bf0a-493d-a0f0-664c4ae31008)

**Comment by NyxDaylight** [comment]
Would be nice for this be re-visited. As mentioned above, it would be easier to just grab it from KDE or Gnomes app store rather than having to download it from wooting's site.

**Comment by pollux78** [comment]
> Would be nice for this be re-visited. As mentioned above, it would be easier to just grab it from KDE or Gnomes app store rather than having to download it from wooting's site.

Yep + free marketing as it will be shown each time on both flathub and the GUI stores on every single major desktop environment on Linux lol
